.gitignore

```diff
@@ -11,6 +11,7 @@ gutenberg.zip
 phpcs.xml
 yarn.lock
 /wordpress
+.vscode
```

Comment:
Historically we've advocated that this be included in a developer's own [global `.gitignore`](https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) to avoid a proliferation of user-specific entries in this file, but this has definitely come up on multiple occasions and I'm curious to potentially consider again (for developer experience' sake) whether to make exceptions at least for the most common cases.

cc @gziolo

Reply:
It's a bit outside the scope of what you're trying to accomplish here, and not of huge consequence one way or the other, but perhaps a topic worth covering in tomorrow's [weekly JavaScript chat](https://make.wordpress.org/meetings/).